ambry-replication/src/main/java/com.github.ambry.replication/ReplicaThread.java

```diff
@@ -69,6 +74,8 @@
 class ReplicaThread implements Runnable {
 
   private final Map<DataNodeId, List<RemoteReplicaInfo>> replicasToReplicateGroupedByNode;
+  private final Set<PartitionId> replicatedPartitions = new HashSet<>();
```

Comment:
Not sure I understand - why is that preferred? That seems like the name of a `Map`